nodes/src/nodes/llm_vision_gemini/IInstance.py

```
@@ -0,0 +1,115 @@
+                question.addQuestion(self.IGlobal._chat._prompt)
+
+                try:
+                    answer = self.IGlobal._chat.chat(question)
+                    answer_text = answer.getText()
+                    self._cached_answer = answer_text
+                    self.instance.writeText(answer_text)
+                except Exception as e:
+                    warning(f'Gemini Vision: inference failed for image frame: {e}')
+
+            self.image_data = None
+            return self.preventDefault()
```

Ruff (0.15.10)

[warning] Missing explicit `return` at the end of function able to return non-`None` value

Add explicit `return` statement

(RET503)